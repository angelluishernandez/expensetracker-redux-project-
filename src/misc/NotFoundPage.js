import React from "react";
import { Link } from "react-router-dom";

const NotFoundPage = () => (
	<div className="NotFoundPage">
		NotFoundPage - <Link to="/">Go Home</Link>
	</div>
);

export default NotFoundPage;
