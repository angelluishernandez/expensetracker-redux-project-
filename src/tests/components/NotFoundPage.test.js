import React from "react";
import { configure } from "enzyme";
import { shallow } from "enzyme";
import expenses from "../fixtures/expenses";
import Adapter from "enzyme-adapter-react-16";
import toJSON from "enzyme-to-json";
import NotFoundPage from "../../misc/NotFoundPage"


configure({ adapter: new Adapter() });

test("should render NotFoundPage correctly", () => {
	const wrapper = shallow(<NotFoundPage />);
	expect(toJSON(wrapper)).toMatchSnapshot();
});
