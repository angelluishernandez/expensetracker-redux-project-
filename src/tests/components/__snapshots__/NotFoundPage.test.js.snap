// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFoundPage correctly 1`] = `
<div
  className="NotFoundPage"
>
  NotFoundPage - 
  <Link
    to="/"
  >
    Go Home
  </Link>
</div>
`;
